domains/proxmox.portkvm.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "windows11-on-web",
+        "email": "idkalone2019@gmail.com"
+    },
+    "record": {
+        "CNAME": "e4caf830-23f7-44d1-9628-a0cabce7b6dd.cfargotunnel.com"
```

Comment:
I cannot reach your website
"This site can’t be reached"

Reply:
My proxmox was offine bro.

Reply:
@Stef-00012